web/client/utils/MapUtils.js

```diff
@@ -380,7 +380,7 @@ function saveMapConfiguration(currentMap, currentLayers, currentGroups, currentB
     return {
         version: 2,
         // layers are defined inside the map object
-        map: assign({}, map, {layers: formattedLayers, groups, backgrounds, text_search_config: textSearchConfig, bookmark_search_config: bookmarkSearchConfig},
+        map: assign({}, map, {layers: formattedLayers, groups, backgrounds, text_search_config: textSearchConfig, bookmark_search_config: bookmarkSearchConfig, previousGroups: currentGroups},
```

Comment:
Hi @jstrR why you need this previousGroups to be saved in the map configuration?

Reply:
@MV88 to get the previous position of nested groups inside heading group